### PR TITLE
Controller/init search frame

### PR DESCRIPTION
**Pull request by MelissaRoman06**
Clean code. setVisible removed from Controller class

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 1]
```diff
@@ -52,9 +50,22 @@
...
+        //Initializes an instance of ButtonListener class
+        ButtonListener buttonListener = new ButtonListener(volumeLevel);
+        //Sends all required arguments to buttonListener's listen method
+        buttonListener.listen(playButton, stopButton, pauseButton, player, fileToBePlayed, progressBar, volumeSlider);
+    }
+
+    /**
+     * Initializes a PlayerJFrame object with all panels, labels and sliders required
+     */
+    private void initComponent(String path){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: please put a space before "{"

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 2]
```diff
@@ -65,15 +76,20 @@ public PlayerJFrame(String path) {
         volumeSlider = new JSlider(JSlider.VERTICAL, 0, 100, volumeLevel);
         player = new EmbeddedMediaPlayerComponent();
         fileToBePlayed = new File(path);
+    }
 
+    /**
+     * Set all PlayerJFrame components
+     */
+    private void initSetting(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: please put a space before "{"

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 3]
```diff
@@ -52,9 +50,22 @@
...
     /**
-     * Initializes a PlayerJFrame object with all panels, labels and sliders required
+     * Initializes a PlayerJFrame object
      */
     public PlayerJFrame(String path) {
+        initComponent(path);
+        initSetting();
+
+        //Initializes an instance of ButtonListener class
+        ButtonListener buttonListener = new ButtonListener(volumeLevel);
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: create the object in initComponent

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 4]
```diff
@@ -52,9 +50,22 @@
     }
 
     /**
-     * Initializes a PlayerJFrame object with all panels, labels and sliders required
+     * Initializes a PlayerJFrame object
      */
     public PlayerJFrame(String path) {
+        initComponent(path);
+        initSetting();
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: delete the empty line

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: outdated

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/PlayerJFrame.java]: done

File: src/main/java/com/fundation/logic/view/PlayerJFrame.java [hunk 5]
```diff
@@ -41,6 +41,7 @@
     private int volumeLevel;
     private EmbeddedMediaPlayerComponent player;
     private File fileToBePlayed;
+    ButtonListener buttonListener;
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/PlayerJFrame.java]: put private at object

**Pull request merged**